### PR TITLE
feat: fe package upgrade

### PR DESCRIPTION
**Pull request by shaddollxz**

### CODE REVIEW

**Comment by shaddollxz** [comment]
测试： https://github.com/shaddollxz/version-upgrade-test/pulls?q=is%3Apr+is%3Aclosed

File: fe-version-upgrade/action.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      if: ${{ inputs.scope }}
+      uses: peter-evans/create-pull-request@v6
+      with:
+        branch: chore/bot/update-${{ steps.get-app-info.outputs.name }}-version
+        title: 'chore(${{ inputs.scope }}): update app ${{ steps.get-app-info.outputs.name }} to v${{ inputs.version }}'
+        commit-message: 'chore(${{ inputs.scope }}): update app ${{ steps.get-app-info.outputs.name }} to v${{ inputs.version }}'
+        base: main
+
+    - name: 'Create Pull Request'
+      if: ${{ ! inputs.scope }}
```
Comment (fahchen) [inline fe-version-upgrade/action.yml:57]:
```suggestion
      if: ${{ !inputs.scope }}
```

File: fe-version-upgrade/action.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+name: 'FE Package Version Update'
+description: 'auto upgrade package.json version'
+
+inputs:
+  version:
+    description: 'Version updated to, greater than or equal to the current version'
+    required: true
```
Comment (fahchen) [inline fe-version-upgrade/action.yml:7]: 这个做成选项吧，不做成具体的版本 https://github.com/Byzanteam/jet-actions/blob/f267cf820e253bfca53464c0df2fe25a98c0f427/denoified-actions/elixir/bump_version/action.yml#L10-L13

File: fe-version-upgrade/action.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        base: main
+
+    - name: 'Create Pull Request'
+      if: ${{ ! inputs.scope }}
+      uses: peter-evans/create-pull-request@v6
+      with:
+        branch: chore/bot/update-${{ steps.get-app-info.outputs.name }}-version
+        title: 'chore: update app ${{ steps.get-app-info.outputs.name }} to v${{ inputs.version }}'
+        commit-message: 'chore: update app ${{ steps.get-app-info.outputs.name }} to v${{ inputs.version }}'
+        base: main
```
Comment (fahchen) [inline fe-version-upgrade/action.yml:63]: 单独给一步来计算 title 和 commit 等信息吧

**Review by fahchen — CHANGES_REQUESTED** [review]
不是特别对，你参考 https://github.com/Byzanteam/jet-actions/tree/main/denoified-actions/elixir/bump_version 来实现吧

**Pull request closed without merging**